sql-plugin/src/main/311until320-all/scala/com/nvidia/spark/rapids/shims/BatchScanExecMeta.scala

```diff
@@ -16,12 +16,19 @@
 
 package com.nvidia.spark.rapids.shims
 
-import org.apache.spark.sql.catalyst.InternalRow
-import org.apache.spark.sql.catalyst.plans.physical.KeyGroupedPartitioning
+import com.nvidia.spark.rapids._
```

Comment:
Not sure why it was marked as rename of the file. I thought it should be delete of KeyGroupedPartitioningShim.scala and addition of BatchScanExecMeta.scala

Reply:
No worries.  This can happen on small files since a large percentage of the small file remained unchanged.